File: week1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
     height = get_int("Height: ");
    } while (!(height > 0 && height < 9));

    int spaces, points;

    for (int i = 0; i < height; i++) {
        spaces = height - i - 1;
        points = i + 1;
        for (int j = 0; j < spaces; j++) {
            printf(" ");
        }
        for (int k = 0; k < points; k++) {
            printf("#");
        }
        printf("\n");
    }


}

File: week6/mario/more/mario.py
divide_spaces = 2

def main():
    height = get_positive_int()
    for i in range(height):
        spaces = height - i -1
        points = i + 1

        print(' ' * spaces, end='')
        print('#' * points, end='')
        print(' ' * divide_spaces, end='')
        print('#' * points)


def get_positive_int():
    while True:
        try:
            result = int(input("Heigh: "))
            if result < 1 or result > 8:
                raise Exception
            return result
        except Exception as e:
            print("Please enter positive integer between 1 and 8 inclusive!")

main()